### PR TITLE
Upgrade React-Native version to 0.50.4

### PR DESCRIPTION
**Pull request by lomudi**
Upgrading the React-Native version to 0.50.4 to align with our latest SDKs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,11 +15,10 @@
   "bugs": {
     "url": "https://github.com/applicaster/Zapp-React-Native-Utils-JS/issues"
   },
-  "homepage":
-    "https://github.com/applicaster/Zapp-React-Native-Utils-JS#readme",
+  "homepage": "https://github.com/applicaster/Zapp-React-Native-Utils-JS#readme",
   "peerDependencies": {
-    "react": "^16.0.0-alpha.12",
-    "react-native": "^0.47.0"
+    "react": "^16.0.0",
```
Comment (Budaa) [inline package.json]: We can use react `16.2.0` as that's the one we added to sdk.

Comment (guykogus) [inline package.json]: We're using `^16.2.0`

**Review by guykogus — CHANGES_REQUESTED** [review]

**Review by Budaa — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Budaa) [reply package.json]: Damn! You were faster.

**Comment by lomudi** [comment]
@Budaa @guykogus i got it... make it 16.2.0

**Review by Budaa — APPROVED** [review]
💯

**Review by guykogus — APPROVED** [review]

**Pull request merged**